rotkehlchen/db/settings.py

```diff
@@ -122,6 +122,20 @@ class DBSettings(NamedTuple):
     non_syncing_exchanges: List[ExchangeLocationID] = []
     cost_basis_method: CostBasisMethod = DEFAULT_COST_BASIS_METHOD
 
+    def serialize(self) -> Dict[str, Any]:
```

Comment:
Need the `serialize()` method?
The endpoint uses `process_result` which I used initially, then Lefteris had a different idea. Although he thought calling `._as_dict()` method was sufficient to serialize it.

Reply:
Hm, wow, alright. Asked this question because adding a new method to `DBSettings` sounds like a big change. But if Our God Lefteris likes it, then should be fine :smile:

Reply:
Yeah ... but did you just copy paste the one from `ModifiableDBSettings`?

They are very similar. You need to make sure common code is abstracted and no code duplication exists.